packages/beacon-node/src/api/impl/lodestar/index.ts

```diff
@@ -218,6 +226,81 @@ export function getLodestarApi({
         meta: {executionOptimistic, finalized, version: fork},
       };
     },
+
+    async getAttesterSlashingsFromBlocks({block_ids}) {
+      const attesterSlashings: AttesterSlashingList = [];
+      const validatorIndicesSeen = new Set<ValidatorIndex>();
+
+      const validatorsAttestations = new Map<ValidatorIndex, IndexedAttestation[]>();
+
+      let fork = chain.config.getForkSeq(chain.clock.currentSlot);
+
+      const blockPromises = block_ids.map((blockId: BlockId) => getBlockResponse(chain, blockId));
```

Comment:
We should provide actual blocks, not references to blocks. Our node might not have all the data we need, eg. in case of Holesky the blocks with slashable attestations were from 2 different / incompatible forks